newsSummaryView.test.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fs = require('fs');
+const NewsSummaryModel = require('./newsSummaryModel');
+const NewsSummaryView = require('./newsSummaryView');
+
+let newsModel;
+let newsView;
+let fakeApi;
+
+beforeEach(() => {
+  document.body.innerHTML = fs.readFileSync('./index.html');
+
+  fakeApi = { getHeadlines: (callback) => callback(
+    [{
+        webTitle: 'Your news today',
+        webUrl: 'https://examplewebsite.com',
+        fields: {
+          thumbnail: 'https://image.jpg'
+        } 
+      }]
+    )
+  };
+
+  newsModel = new NewsSummaryModel();
+  newsView = new NewsSummaryView(newsModel, fakeApi);
```

Comment:
Asme - there are global variables. You may want to use `let` or `const`. Also, adding `"use strict"` at the top of your file will stop you from having globals.
Check https://www.w3schools.com/js/js_strict.asp#:~:text=Strict%20mode%20changes%20previously%20accepted,accidentally%20create%20a%20global%20variable.

Reply:
Same reason here. I declared the variables outside of the beforeEach block so then they could be visible in every it block. I guess next time it will be better to keep them within the it blocks. But what if I have several it blocks using all the same variables?